### DISCUSSION TITLE
How to show a tooltip inside a table

### DISCUSSION THREAD

**Discussion by LauGroupie95** [buildo/bento-design-system#312] [Q&A] [answered]
Hi @veej and @gabro,
I tried using the tableColumn.textWithIcon to create the following layout (see image), but now I'm not able to add any tooltip associated to the icon. Please, can you help me?

![image](https://user-images.githubusercontent.com/63547770/181000457-3968950c-0227-48bf-b143-043c0e10322a.png)

Here's the playroom:
https://playroom.bento.buildo.io/#?code=N4Igxg9gJgpiBcIA8AVAhgIwDYwDoDsACQqNAFzQF5gBtA444eh4yAV3zICcBPeQpkRYMAlpHz8AkuMn4AZhC4BbciIBuMADTNhhMjAAeZfrhABVAMqntQlgF8b9x4QC6d5pCxsl%2BAM7U6Wz1MHABhCC8fADp9IwB1ETIAC2kIfAAKQV0kmDRYLgAZTBgsExBwjm4eax0GNDAwGF9fRTL2Tl4aoOIxNIAFCF9EkTSyrhEAcySyLvsASmc3AgB6AD4CEE0QAHcRKGTfBBoARgAWU4AGNyA

I tried also using a tableColumn.custom but if I use something like 
< Inline space... >
     < Body size... > content < / Body > 
     < Tooltip ...  / >
< / Inline >
it says that multiple children are not accepted, so maybe that's not the right way to proceed.

Thank you in advance

**Comment by gabro** [comment, marked as answer]
Hi @LauGroupie95, it's a good point, `textWithIcon` doesn't currently allow that. We may add it (I'll discuss this with @veej and @Phyele) but in the meantime you can work around this by using a custom column (via `tableColumn.custom`)

**Reply by LauGroupie95** [reply]
Ok, at the moment I solved this way, thank you

```
const trigger: TooltipProps["trigger"] = (ref, props) => (
        <Box ref={ref} {...props} display="inline-block">
            <IconInformative size={12} />
        </Box>
    );

<Table
  data={[
    {
      country: {
        body: "US",
        tooltipContent: "United States",
      },
    },
  ]}
  columns={[
    tableColumn.custom({
      headerLabel: "Country",
      accessor: "country",
      disableSortBy: true,
      width: "fill-available",
      Cell: (p: any) => (
        <Inline space={8} alignY="center">
          <Body size="medium">{p.cell.value.body}</Body>
          <Tooltip content={p.cell.value.tooltipContent} trigger={trigger} />
        </Inline>
      ),
    }),
  ]}
/>

```

**Reply by gabro** [reply]
👍 thats looks good! I'll ping you here if we add `tooltip` as an optional prop to `textWithIcon`

**Reply by gabro** [reply]
@LauGroupie95 we're adding this in the next release, follow the PR here https://github.com/buildo/bento-design-system/pull/316

**Reply by gabro** [reply]
Released as 0.12.1

**Reply by LauGroupie95** [reply]
Wonderful, thank you!
